system/cfme-setup.sh

```diff
@@ -31,6 +31,17 @@ cat <<'EOF' > /etc/httpd/conf.d/ssl.conf
 # upgraded.
 EOF
 
+# Incorporate the iptable updates
+if [[ -f /etc/sysconfig/iptables.miq ]]; then
+  /sbin/iptables-save > /etc/sysconfig/iptables.SYSTEM
+  /sbin/iptables-restore /etc/sysconfig/iptables.miq
+  /sbin/service iptables save
+  # Expect no more than the four datestamp differences.
+  if [[ $(/usr/bin/diff -Ew -U 0 /etc/sysconfig/iptables /etc/sysconfig/iptables.miq  | /bin/grep "^@@" | /usr/bin/wc -l) -gt 4 ]]; then
```

Comment:
So, yes, I agree it's useful but once we have it working, it's not needed.  I can't imagine something changing once we have this working that would cause it to fail again.  It smells like test code until we have it working, and then it can be removed.